core/tests/webdriverio_desktop/learnerDashboard.js

```diff
@@ -141,7 +141,7 @@ describe('Learner dashboard functionality', function() {
     await users.logout();
   });
 
-  it('should add exploration to play later list', async function() {
+  it('should add and remove exploration to play later list', async function() {
     var EXPLORATION_FRACTION = 'fraction';
     var EXPLORATION_SINGING = 'singing';
     var CATEGORY_MATHEMATICS = 'Mathematics';
```

Comment:
Please use `let` instead of `var`

Reply:
@Lawful2002 Should I replace all instances of "var" with "let" throughout the files?

Reply:
Yes

Reply:
@Lawful2002I tried replacing all occurrences of 'var' in the file, but it is leading to several errors. As a result, I have only substituted 'var' in this PR and its associated areas. Would it be advisable to create an issue to handle the replacement of 'var' in all files? Making all `var` changes in this PR might not be a good idea as it includes adding a test. If errors arise after the PR is merged, we would need to revert the entire PR. Therefore, I believe tackling this in parts would be a better approach. What are your thoughts on this?

Reply:
@HardikGoyal2003 Just a note, it would probably help the reviewer if you explained what the errors are.

Reply:
@seanlip @Lawful2002 One error I encountered was the duplicate variable name error. In JavaScript, `var` allows variables to be reassigned, while `let` only allows declaration once. This led to variables being redefined in multiple places and used across different files. To resolve this, we need to change the variable names when redeclaring or declare them only once in all e2e suite files.

Reply:
Hmm, I see, but I don't understand why we need to declare a variable twice in a given scope. I took a look at the screenshot you posted below, and I think I have found a fix. In `learnerDashboard.js` on line `346` we declare `url`, but within the same scope on line `365`, we redefine it. The fix should be simple: on line `346` we do `let url = await browser.getUrl();` and on line `365` we just reassign the url like so: `url = await browser.getUrl();`. This should hopefully fix the problem you posted.
That said, as this change is outside the scope of this PR, we can ignore changing declarations in this PR.

Reply:
Done! Thanks!